core/contracts/common/interfaces/AccessControlInterface.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.5.0;
+
+// Sourced from proposed OpenZeppelin implementation of ERC20Mintable with v3 contracts
+// - description: https://forum.openzeppelin.com/t/redesigning-access-control-for-the-openzeppelin-contracts/2177/24
+// - source code: https://gist.github.com/nventuro/ac42bd936f678ebf55c4c0c4a4ee4c72
+
+interface IAccessControl {
```

Comment:
Same comment as above, this is very similar to our MultiRole contract -- IMO, we shouldn't reinvent the wheel here.

Reply:
I agree with these and I will revert to using Multirole.